.travis.yml

```diff
@@ -13,15 +13,15 @@ services:
 # Ensure docker is updated
 before_install:
```

Comment:
I think we can actually remove this and just use 

```
# Enable the docker service
services:
- docker
```

Reply:
Yea, the only thing it does now is upgrade docker.

Reply:
We could also add `cache: pip` to cache packages installed via pip